custom_components/xiaomi_miio_airconditioningcompanion/manifest.json

```
@@ -5,7 +5,7 @@
   "issue_tracker": "https://github.com/EugeneLiu/xiaomi_airconditioningcompanionMCN02/issues",
   "version": "0.1.7",
   "requirements": [
-    "construct==2.9.45",
+    "construct>=2.10.68",
```

Should we at least pin it at major version 2?
```suggestion
    "construct~=2.10",
```